src/openrct2/network/network.cpp

```diff
@@ -307,6 +307,15 @@ bool Network::BeginServer(uint16 port, const char* address)
 	player->Group = 0;
 	player_id = player->Id;
 
+	if (network_get_mode() == NETWORK_MODE_SERVER) {
+		// Add SERVER to users.json and save.
+		NetworkUserManager *userManager = &gNetwork._userManager;
```

Comment:
The second looks better, I would have gone for the top if it was not in the same class.